src/JsonSchema/DefinitionNameFactory.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\JsonSchema;
+
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\Metadata\Util\ResourceClassInfoTrait;
+use Symfony\Component\Serializer\Normalizer\AbstractNormalizer;
+
+/**
+ * @internal
+ */
+final class DefinitionNameFactory implements DefinitionNameFactoryInterface
+{
+    use ResourceClassInfoTrait;
+
+    public function __construct(private ?array $distinctFormats)
+    {
+    }
+
+    /**
+     * @param class-string $className
```

Comment:
Leave this as is or `@inheritdoc`?

Reply:
you can remove inheritdoc is automatic now in most of the tools